doc/classes/Node.xml

```diff
@@ -1076,8 +1076,9 @@
 			[b]Note:[/b] In the editor, nodes not owned by the scene root are usually not displayed in the Scene dock, and will [b]not[/b] be saved. To prevent this, remember to set the owner after calling [method add_child].
 		</member>
 		<member name="physics_interpolation_mode" type="int" setter="set_physics_interpolation_mode" getter="get_physics_interpolation_mode" enum="Node.PhysicsInterpolationMode" default="0">
-			Allows enabling or disabling physics interpolation per node, offering a finer grain of control than turning physics interpolation on and off globally. See [member ProjectSettings.physics/common/physics_interpolation] and [member SceneTree.physics_interpolation] for the global setting.
-			[b]Note:[/b] When teleporting a node to a distant position you should temporarily disable interpolation with [method Node.reset_physics_interpolation].
+			The physics interpolation mode to use for this node. Only effective if [member ProjectSettings.physics/common/physics_interpolation] or [member SceneTree.physics_interpolation] is [code]true[/code].
+			This allows enabling or disabling physics interpolation on a per-node basis, regardless of its parents' physics interpolation mode. By default, the node inherits the physics interpolation mode from its parent. Certain node types like [VehicleWheel3D] have physics interpolation disabled by default, as they rely on their own custom solution for physics interpolation out of the box.
+			[b]Note:[/b] When teleporting a node to a distant position, you should temporarily disable interpolation with [method Node.reset_physics_interpolation] [i]after[/i] moving the node.
```

Comment:
Perhaps this means the same **exact** thing, but to me this feels more like an advice:
```suggestion
			[b]Note:[/b] When teleporting a node to a distant position, it's recommended to temporarily disable interpolation with [method Node.reset_physics_interpolation] [i]after[/i] moving the node.
```
Either that, or you'd have to explain why "you should", but that's better done in `reset_physics_interpolation`